assets/app/scripts/app.js

```diff
@@ -160,6 +160,26 @@ angular
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
   .constant("AUTH_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).auth))
+  .factory("ProxyPod", function(API_CFG) {
+    return function(namespace, podId, port) {
+      var kubeProxyURI = new URI("https://" + API_CFG.k8s.hostPort + API_CFG.k8s.prefix);
+      var apiVersion = API_CFG.k8s.version || 'v1beta3';
+      if (port) {
+        podId = podId + ':' + port;
+      }
+      switch(apiVersion) {
+        case 'v1beta3':
+          kubeProxyURI.path(UrlHelpers.join(kubeProxyURI.path(), 'v1beta3/proxy/namespaces', namespace, 'pods', podId));
```

Comment:
web users are unlikely to have the permissions they need to make an API call to `/proxy/namespaces/{ns}/pods/{pod}`...